core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -341,6 +341,10 @@ public static FunctionExpression day_of_week(
     return compile(functionProperties, BuiltinFunctionName.DAY_OF_WEEK, expressions);
   }
 
+  public static FunctionExpression extract(Expression... expressions) {
+    return compile(FunctionProperties.None, BuiltinFunctionName.EXTRACT, expressions);
```

Comment:
The function requires `FunctionProperties` unfortunately, you can post signature with FP only or both.

Reply:
See signature for `equal` for example.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/commit/d15c26ba044d8db36dad2dcf197757404aeb8ece